website/src/panel-layout.tsx

```diff
@@ -78,8 +87,39 @@ export const PanelLayout = (p: {
 
   let panel = null
 
+  // Logic to display comment panel if the button is pressed
+  const handleComment = (e: React.MouseEvent<HTMLButtonElement>) => {
+    e.preventDefault()
+
+    if (commentsPanel === true) {
+      setCommentsPanel(false)
+    } else {
+      setCommentsPanel(true)
+    }
+  }
+
```

Comment:
This if block can be cleaned up using the power of Boolean logic.

Heres what I think youre trying to do:
When commentsPanel is true, do setCommentsPanel(false).
When commentsPanel is false, do setCommentsPanel(true).

Lets see what would happen if we tried using negation here:
```TypeScript
setCommentsPanel(!commentsPanel)
```

What does this do? Lets find out:
```PlainText
1. assume commentsPanel is true
setCommentsPanel(!commentsPanel) (Given)
setCommentsPanel(!true) (substitute)
setCommentsPanel(false) (negation)

2. assume commentsPanel is false
setCommentsPanel(!commentsPanel) (Given)
setCommentsPanel(!false) (substitute)
setCommentsPanel(true) (negation)
```
Its a match! Please replace this whole if block with `setCommentsPanel(!commentsPanel)`.